README.md

```diff
@@ -65,7 +65,7 @@ All commands are run from the root of the project, from a terminal:
 
 DDev already has all the dependencies included.
 
-1. Run `ddev start && ddev npm install` set up the project’s dependencies.
+1. Run `ddev start && ddev npm cache clean --force  && ddev npm install ` set up the project’s dependencies.
 3. `ddev npm run build` can be found at the base URL and `ddev npm run dev` is found at https://<projectname>.ddev.site:4321. Dev has Vite HMR(hot module reloading) among other features. The site will automatically refresh as you work on it, displaying errors in the relevant console (terminal or browser).
```

Comment:
3 → 2, fix spacing, reword slightly for clarity:

```suggestion
2. `ddev npm run build` can be found at the base URL and `ddev npm run dev` is found at https://<projectname>.ddev.site:4321. The dev server has Vite HMR (hot module reloading) among other features. The site will automatically refresh as you work on it, displaying errors in the relevant terminal or browser console.
```

What do you mean by `ddev npm run build` can be found at the base URL?

I know that running the `build` command generates a static version of the site into the `dist/` directory, which you can run the `preview` command to serve, but the wording of this seems sort of ambiguous. Similarly with “`ddev npm run dev` is found at”—the command isn’t found at a browser address, but running it starts a server you can access from the browser at that address. I know you’re aware of this, but the wording leaves it unclear for someone figuring it out.